src/main/java/com/linkedin/kmf/services/MultiClusterTopicManagementService.java

```diff
@@ -182,13 +199,13 @@ public void run() {
 
     void maybeCreateTopic() throws Exception {
       if (_topicCreationEnabled) {
-        _topicFactory.createTopicIfNotExist(_zkConnect, _topic, _replicationFactor, _partitionsToBrokerRatio, new Properties());
+        _topicFactory.createTopicIfNotExist(_zkConnect, _topic, _replicationFactor, _minPartitionsToBrokersRatio, new Properties());
```

Comment:
@smccauliff Make sense. This is definitely worth doing. I will submit a separate patch either today or tomorrow to address this problem.